packages/pwa-kit-extension-sdk/src/react/classes/ApplicationExtension.ts

```diff
@@ -43,17 +46,26 @@ export class ApplicationExtension<
 
     /**
      * Called during server rendering and client application initialization. This method allows
-     * you to modify the routes of the base application, typically used to add new routes pointing
-     * at page components added by your application extension.
+     * you to add new routes, typically routes pointing at page components added by your application extension.
      *
      * @protected
-     * @param routes - The list application routes currently loaded.
-     * @returns routes - The modified application routes.
+     * @returns new routes to be added
      */
-    public extendRoutes(routes: RouteProps[]): RouteProps[] {
-        return routes
+    public getRoutes(): RouteProps[] {
```

Comment:
What is your plan here with respect to arguments? Is there a reason why we aren't passing anything. It might be nice to have symmetry with getRoutesAsync

Reply:
I removed the params as a response to this comment: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2308#discussion_r2006197453

_But_ now I'm thinking to revert the change 😅 Sorry @hajinsuha1 for me being easily swayed in this case.

Inspired by [Node's filesystem API](https://nodejs.org/api/fs.html#promise-example), I now think that it makes more sense to have symmetry like what Ben said. There isn't a good usecase right now for passsing in the same params to `getRoutes`. However, it's one less thing to remember for the developers. They can remember 1 method signature for both.